misc/python/materialize/mzcompose/composition.py

```diff
@@ -490,7 +491,16 @@ def sql_connection(
     ) -> Connection:
         """Get a connection (with autocommit enabled) to the materialized service."""
         port = self.port(service, port) if port else self.default_port(service)
-        conn = pg8000.connect(host="localhost", user=user, password=password, port=port)
+        ssl_context = ssl.create_default_context()
+        ssl_context.check_hostname = False
+        ssl_context.verify_mode = ssl.CERT_NONE
+        conn = pg8000.connect(
+            host="localhost",
+            user=user,
+            password=password,
+            port=port,
+            ssl_context=ssl_context,
```

Comment:
I'm worried this will break all other callers that have ssl disabled. Am waiting for tests to see.